easybuild/easyconfigs/n/nanopolish/nanopolish-0.10.2-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,46 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+# Fred Hutchenson Cancer Research Center
+#
+#  June 13, 2017 0.7.1
+#  Dec 31, 2018, 0.10.2
+easyblock = 'MakeCp'
+
+name = 'nanopolish'
+version = '0.10.2'
+versionsuffix = '-Python-2.7.15'
```

Comment:
@fizwit Please use `%(pyver)s` rather than hardcoding the `Python` version here, I had to fix this in my PR too (see a1d1ee684)